packages/templating/plugin/html_scanner.js

```diff
@@ -169,8 +171,15 @@ html_scanner = {
           templateDotNameLiteral + ", " + renderFuncCode + ");\n";
       } else {
         // <body>
-        if (hasAttribs)
-          throwParseError("Attributes on <body> not supported");
+        if (hasAttribs) {
+          for(var attr in attribs) {
+            if (this.bodyAttributes.indexOf(attr) !== -1) {
+              console.log('WARNING: tag ' + attr + ' already defined on body.');
```

Comment:
Yep, an error seems good to me too and there really isn't any control on the order.
I'll change that..